packages/amplify-category-storage/commands/storage.js

```diff
@@ -30,6 +30,10 @@ module.exports = {
         name: 'remove',
         description: `Removes ${featureName} resource from your local backend. The resource is removed from the cloud on the next push command.`,
       },
+      {
```

Comment:
I don't think we support console for this category yet....

Reply:
there's is such a command though, but it prints out to be implemented. 

    context.print.info(`to be implemented: ${category} ${subcommand}`);